webapps/pom.xml

```diff
@@ -0,0 +1,269 @@
+          <artifactId>maven-antrun-plugin</artifactId>
+          <version>1.8</version>
+        </plugin>
+        <plugin>
+          <groupId>org.codehaus.mojo</groupId>
+          <artifactId>build-helper-maven-plugin</artifactId>
+          <version>1.9.1</version>
+        </plugin>
+
+        <plugin>
+          <groupId>org.eclipse.jetty</groupId>
+          <artifactId>jetty-maven-plugin</artifactId>
+          <version>${version.jetty}</version>
+        </plugin>
+
+        <plugin>
+          <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-war-plugin</artifactId>
+          <configuration>
+            <webResources>
+              <resource>
+                <filtering>true</filtering>
+                <directory>${web.resources.override}</directory>
+              </resource>
+              <resource>
+                <filtering>false</filtering>
+                <directory>target/webapp</directory>
+                <excludes>
+                  <exclude>**/*.html</exclude>
+                </excludes>
+              </resource>
+              <resource>
+                <filtering>true</filtering>
+                <directory>target/webapp</directory>
+                <!-- no filtering for favicon because linux destroys images through carriage return -->
+                <includes>
+                  <include>**/*.html</include>
+                </includes>
+              </resource>
+            </webResources>
+            <!-- exclude development resources (task forms etc.) -->
+            <packagingExcludes>
+              develop/**
+            </packagingExcludes>
+          </configuration>
+        </plugin>
+        <plugin>
+          <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-jar-plugin</artifactId>
+          <executions>
+            <execution>
+              <id>create-jar</id>
+              <goals>
+                <goal>jar</goal>
+              </goals>
+              <phase>package</phase>
+              <configuration>
+                <classifier>classes</classifier>
+              </configuration>
+            </execution>
+          </executions>
+        </plugin>
+        <plugin>
+          <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-assembly-plugin</artifactId>
+          <executions>
+            <execution>
+              <id>zip-frontend-sources</id>
+              <phase>generate-resources</phase>
+              <goals>
+                <goal>single</goal>
+              </goals>
+              <configuration>
+                <skipAssembly>${skip-zip-frontend-sources}</skipAssembly>
+                <descriptors>
+                  <descriptor>assembly.xml</descriptor>
+                </descriptors>
+              </configuration>
+            </execution>
+          </executions>
+        </plugin>
+      </plugins>
+    </pluginManagement>
+  </build>
+
+  <modules>
+    <module>assembly</module>
+  </modules>
```

Comment:
❓ Shouldn't this contain both modules? Otherwise, we only build one module on CI.

Reply:
We add the`assembly-jakarta` module only for JDK 11 or higher, see below. All the Jakarta libraries require a min JDK of 11.